### ISSUE TITLE
Sprint Goals Apr 20 - May 04

### ISSUE THREAD

**Issue by jmann2817** [CDCgov/data-exchange-upload#84]
**Sprint Goals:**
Release management/documentation/testing is focus for Matt/Chase/et al
- [x] Develop test plan 
- [ ] Review Update DEX Upload API document for supplemental API (planned for 1.1)
- [ ] Test tus file-hook for post-terminate
- [ ] Standup staging environment
- [ ] Verification/QA staging environment

**Stretch** 
- [x] onboard two people from PulseNet.  PulseNet team was able to successfully call the API upload. 
  - [x] Boris will add Cosmin as a SAMS admin.
  - [x] Provide onboard documentation.
  - [x] SAMS admin will create SAMS accounts for two PulseNet people.  
  - [ ] What configurations changes have to be made to support PulseNet?  Program, event type,  L1 metadata
  - [x] Create front-end browser application (client example) for PulseNet

Onboarding new program users  
TL;DR process...

A) Onboarding to existing
1. Add SAMS account
2. Add to Activity
3. End-user gets system account
4. Use Examples

B) Only for new programs
1. Update metadata at L1
2. Optional: Update L2 metadata
3. Configure scopes

SAMS Auth flow
https://lucid.app/lucidchart/185d934a-9653-4777-aa54-b9697c02fcb9/edit?page=rPyME0onhlEa&invitationId=inv_431bc0e9-cc80-4e06-b7e9-dc218d20fe52#

**Comment by mkrystof** [comment]
Release 1.1 planning removed.  Date and scope of functionality not yet determined.
